project/stac_api/models.py

```diff
@@ -15,6 +17,30 @@
 # all the comments can be deleted later on
 
 
+# after discussion with Chris and Tobias:
+# stac_extension will be populated with default values that are set to be
+# non-editable for the moment. Could be changed, should the need arise.
+# The following - a bit complicated approach - hopefully serves to solve the
+# error:
+# <begin Quote>
+# "*ArrayField default should be a callable instead of an
+# instance so that it's not shared between all field instances.
+# HINT: Use a callable instead, e.g., use `list` instead of `[]`.""
+# <end quote>
+def get_default_stac_extensions():
+    DEFAULT_STAC_EXTENSIONS = { # pylint: disable=invalid-name
```

Comment:
I would stick to the convention here and don't disable the pylint invalid-name. By convention we should have `default_stac_extensions` variable in lower case (upper case is for global constant)

Reply:
OK, I had in mind, that we name constants in capital letters.. I will change that

Reply:
After checking our guideline and also other services, I guess, capitals is correct, as it is a dictionary that represents constant values, e.g. like `ALLOWED_DOMAINS` in `service-qrcode` in the `url.py`.
What do you think?

Reply:
Python code guideline consider upper case for global constant, here the variable is a function scope constant and should be lower, however you could move the variable outside the function scope to be global and keep the upper case notation without having pylint complaining.

Reply:
OK, done. (thought you meant "global" in terms of the whole app)